scrapper.py

```diff
@@ -42,7 +53,18 @@ def find_articles(self):
         """
         Finds articles
         """
-        pass
+        raw_urls_list = []
+        for url in self.seed_urls:
+            response = requests.get(url, headers=headers)
+            print('Making a request...')
+            sleep_interval = random.randrange(2, 5)
+            sleep(sleep_interval)
+            text = response.text
+            link_sample = re.compile(r'<h3><a href="/\d{6}.+"')
```

Comment:
it could be the solution. BUT, we are using **beautiful**soup and there should be a **beautiful** solution for searching all links, right?